zppy/__main__.py

```diff
@@ -29,15 +29,23 @@ def main():
     templateDir = os.path.join(os.path.dirname(__file__), "templates")
 
     # Read configuration file and validate it
-    config = ConfigObj(args.config, configspec=os.path.join(templateDir, "default.ini"))
-    validator = Validator()
-
-    result = config.validate(validator)
-    if result is not True:
-        print("Validation results={}".format(result))
-        raise Exception("Configuration file validation failed")
+    default_config = os.path.join(templateDir, "default.ini")
+    user_config = ConfigObj(args.config, configspec=default_config)
+    _validate_config(user_config)
+    campaign = user_config["default"]["campaign"]
+    if campaign != "custom":
+        campaign_file = os.path.join(templateDir, "{}.cfg".format(campaign))
```

Comment:
If the user wants to create their own config file, does it also have to be in `templateDir` then -- i.e., ``zppy/templates/<my-custom-campaign>.cfg``?

Reply:
No, the user puts things into the usual config file that's passed to `zppy` with `-c`.  There is no added advantage in putting a file in `templateDir` unless it's something to be shared among everyone, like the campaign defaults.